amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpRpcFlowStage.scala

```diff
@@ -53,7 +51,7 @@ final class AmqpRpcFlowStage(settings: AmqpSinkSettings, bufferSize: Int, respon
       private val queue = mutable.Queue[CommittableIncomingMessage]()
       private var queueName: String = _
       private val unackedMessages = new AtomicInteger()
-      private var outstandingMessages = 0
+      private val outstandingMessages = new AtomicInteger()
```

Comment:
As this was actually only touched in async callbacks (and therefore always on the stage thread) it can safely be a "normal" var. I think this is also true for `unackedMessages`, must have been some rework of this stage from an implementation that did modify them concurrently.

Reply:
I've changed them to var.